scripts/linters/general_purpose_linter.py

```diff
@@ -731,9 +731,10 @@ def get_linters(
         verbose_mode_enabled: bool. True if verbose mode is enabled.
 
     Returns:
-        linter: object(linter). Returns linter object.
+        linter: tuple(GeneralPurposeLinter). Returns custom and third_party
```

Comment:
I think you can write the typeinfo as "(GeneralPurposeLinter, None)."

Ditto elsewhere.

Reply:
Done.